### PR TITLE
Framework/Carthage support

### PR DESCRIPTION
**Pull request by EricMuller22**
In-flight/WIP branch to add Carthage support #35 

cc @dfed

### CODE REVIEW

File: Valet.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -7,6 +7,9 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		AC2C2D481B8CE184008F4EE8 /* Valet_iOS_Framework.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = AC2C2D311B8CE184008F4EE8 /* Valet_iOS_Framework.framework */; };
```
Comment (dfed) [inline Valet.xcodeproj/project.pbxproj]: do we want an OS X framework as well? Or does Carthage not do OS X (#newb)?

**Comment by EricMuller22** [comment]
Just started with iOS, set the deployment target to 8.0 - build output here:
screen shot 2015-08-25 at 11 01 21 am

**Comment by dfed** [comment]
Let's make sure we don't break module maps. Cc @mikelikespie, our resident expert on them.

File: Valet.xcodeproj/project.pbxproj (hunk 1 above)
Comment (eliperkins) [inline Valet.xcodeproj/project.pbxproj]: What do you think about making `Valet.framework` should be the output name of the framework here? This file will be nested inside of `Carthage/Builds/iOS` anyway, so the additional naming of iOS is redundant, and `_Framework.framework` is also redundant.

Comment (eliperkins) [inline Valet.xcodeproj/project.pbxproj]: Also, Carthage does OS X!

**Comment by EricMuller22** [comment]
Closing in favor of #42. Thanks @eliperkins!

**Pull request closed without merging**